backend/gke/kubeconfig.sh

```diff
@@ -1,13 +1,23 @@
 #!/usr/bin/env bash
 
-. ./defaults.sh
```

Comment:
I did this so that if someone is not using CI and do a kubeconfig import locally, the script should not fail even if they provide KUBECFG=kubeconfig.yaml, instead of KUBECFG=ClusterReference.yaml.
I will provide a if statement for Clusterreference file processing block:
if "kind: ClusterReference" then process cluster reference file other wise move to ./defaults line

Reply:
If post processing ClusterReference, any gke variable e.g GKE_PROJECT  remains unset, ./default will set it. That is way I kept it after ClusterReference processing. Let me know if there is something I am missing due my lack of understanding with respect to default's exports.

23f1b5891846c284bc7b74a6554059158bf70c51